tests/std/tests/P0896R4_ranges_alg_transform/test.cpp

```diff
@@ -0,0 +1,150 @@
+// Copyright (c) Microsoft Corporation.

```

Comment:
This test needs to be added to `tests/std/test.lst`.

Reply:
Done. 

As they are rather hefty I have splitted them between unary and binary transform